modules/quick/inventory/src/Plugin/QuickForm/Inventory.php

```diff
@@ -494,7 +496,10 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
    */
   public function submitConfigurationForm(array &$form, FormStateInterface $form_state) {
     $this->configuration['asset'] = $form_state->getValue('asset');
-    $this->configuration['units'] = $form_state->getValue('units');
+    $this->configuration['units'] = '';
```

Comment:
I believe this would end up saving an empty string if nothing was provided?

Reply:
True, I suppose it should explicitly set it to `NULL` if that's the new default. I'll change it to `NULL`.

We do need to explicitly set it here otherwise there's no way to "unset" units in configuration.